layouts/partials/head_custom.html

```diff
@@ -15,4 +15,17 @@
 
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Exo:wght@400;700&display=swap" rel="stylesheet">
\ No newline at end of file
+<link href="https://fonts.googleapis.com/css2?family=Exo:wght@400;700&display=swap" rel="stylesheet">
+
+
+{{ if or (eq .Section "specifications") (strings.HasPrefix .Section "specifications/") }}
+  {{ if (strings.HasPrefix .RelPermalink "/zh/")}}
```

Comment:
Couldn't we just use the `.Language.lang` call from https://gohugo.io/methods/page/language/ to handle all current and future languages?

Reply:
Good call. I've updated it to be language agnostic